docs/releases/1.9.txt

```diff
@@ -127,7 +127,9 @@ Management Commands
 Models
 ^^^^^^
 
-* ...
+* Added the :meth:`django.db.models.fields.related.RelatedManager.set()` method
```

Comment:
`RelatedManager.set() <django.db.models.fields.related.RelatedManager.set>` rather then rendering the entire path?